src/components/OrderDetails.tsx

```diff
@@ -302,7 +309,7 @@ export function OrderDetails({
               <InfoTooltip text={Number(limitPrice).toFixed(18)} /> {priceUnit}
             </div>
           </OrderDetailsInformation>
-          {(order?.filledPct || 0) > 0 && (
+          {(Number(order?.stopLossData?.filledPctBpt) || 0) > 0 && (
```

Comment:
why bpt? Did you mean `bps` as in basis points?

Reply:
yes, I will change this